react/components/Divider/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cn from 'classnames'
+
+const propTypes = {
+  orientation: PropTypes.oneOf(['vertical', 'horizontal']),
+  /**
+   * Specifies the dividing line size.
+   */
+  size: PropTypes.number,
+  /**
+   * Specifies the gaps bettwen rows or columns.
+   */
+  gap: PropTypes.number,
+  /**
+   * Specifies the line color.
+   */
+  color: PropTypes.string,
+}
+
+const Divider: React.FC<PropTypes.InferProps<typeof propTypes>> = props => {
+  const size = {
+    [props.orientation === 'horizontal' ? 'height' : 'width']: props.size,
+  }
+  const gap =
+    props.orientation === 'horizontal'
+      ? `mt${props.gap} mb${props.gap}`
```

Comment:
why not `mv${props.gap}` and `mh${props.gap}` for ml and mr below?

Reply:
and you may want to validate props.gap, otherwise user can pass a value like: `gap: '0 my-custom-tachyon-token'` and insert any class he wants there hehe

Reply:
i don't know all tachyons classes yet, but we will not use the gap anymore.